src/Root/routes.js

```diff
@@ -7,10 +7,13 @@ import LandingPage from './LandingPage';
 
 const checkAuth = ({ location: { pathname } }, replace) => {
   const isAuthenticated = !!localStorage.getItem('KIZUNA_USERNAME');
+
   if (isAuthenticated && pathname === '/login') {
     replace('/');
   } else if (!isAuthenticated && pathname !== '/login') {
     replace('/login');
+  } else if (isAuthenticated) {
+    return;
```

Comment:
I just realise why this looks strange to me. Basically, `` /login `` is a public page. However, you nest `` /login `` beneath private page root. Thus you are working around your own auth check code.